docs/metrics.md

```diff
@@ -57,13 +44,14 @@ kubectl create -f manifests/
 kubectl get all -n monitoring
 ```
 
-Delete stack:
+#### Grant permission for prometheus to scrape metrics
 
-```bash
-kubectl delete --ignore-not-found=true -f manifests/ -f manifests/setup
-```
+Go to `ramen/config/hub/default/k8s/kustomizations.yaml`
+and uncomment `../../../prometheus`
+and `metrics_role_binding.yaml` under `Kustomization` section.
+Next is to install and configure ramen.
 
-### Dashboard Access
+#### Dashboard Access
```

Comment:
Point to docs from Prometheus rather than repeating the steps here.

Reply:
updated